lib/single-new-eks-awsnative-fargate-observability-construct/index.ts

```diff
@@ -0,0 +1,70 @@
+import { Construct } from 'constructs';
+import * as blueprints from '@aws-quickstart/eks-blueprints';
+import { ObservabilityBuilder } from '../common/observability-builder';
+import * as eks from 'aws-cdk-lib/aws-eks';
+
+
+export default class SingleNewEksAWSNativeFargateobservabilityConstruct {
+    constructor(scope: Construct, id: string) {
+        // AddOns for the cluster
+        const stackId = `${id}-observability-accelerator`;
+
+        const account = process.env.COA_ACCOUNT_ID! || process.env.CDK_DEFAULT_ACCOUNT!;
+        const region = process.env.COA_AWS_REGION! || process.env.CDK_DEFAULT_REGION!;
+
+        const cloudWatchAdotAddOn = new blueprints.addons.CloudWatchAdotAddOn({
+            deploymentMode: blueprints.cloudWatchDeploymentMode.DEPLOYMENT,
+            namespace: 'default',
+            name: 'adot-collector-cloudwatch',
+            metricsNameSelectors: ['apiserver_request_.*', 'container_memory_.*', 'container_threads', 'otelcol_process_.*'],
+        });
+        
+        const addOns: Array<blueprints.ClusterAddOn> = [
+            new blueprints.addons.VpcCniAddOn(),
+            new blueprints.addons.CoreDnsAddOn({
+                version:"v1.10.1-eksbuild.1",
```

Comment:
Is there a reason why you need to  pass a different version because addon already has `v1.10.1-eksbuild.2` ? Also why is he ConfigurationVaules needed, is that needed for Fargate. Reason why im asking this is because these vaules should be passed as CoreDnspropos to builder in ObservabilityBuilder as blueprints.

Reply:
Fargate cluster fails with coredns v1.10.1-eksbuild.2 with error `Resource handler returned message: "Addon version specified is not supported `